chall02/cdorta-n.py

```diff
@@ -0,0 +1,40 @@
+#/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+import sys
+
+num = len(sys.argv)
+
+if num != 2:
+        print("usage: ./cdorta-n.py <a-zA-Z string>")
```

Comment:
You should extract you error handling message because it's not the only time that you'll need it